### ISSUE TITLE
[SF][Chart - ScatterChart, Donut]: Chart contents are not fully accessible using keyboard/screen reader

### ISSUE THREAD

**Issue by Muthu3982** [UI5/webcomponents-react#7484]
### Describe the bug

None of the web component chart contents/legends are not accessible by keyboard or neither announced by screen reader, but the same chart components in SAP UI5 is accessible via keyboard and screen reader able to announce the chart contents/data properly.

**Web Component Charts Example:**
https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/charts-donutchart--docs

**UI5 Chart Component EXamples:**
https://ui5.sap.com/#/entity/sap.viz.ui5.controls.VizFrame/sample/sap.viz.sample.Donut
https://ui5.sap.com/#/entity/sap.viz.ui5.controls.VizFrame/sample/sap.viz.sample.Line

### Reproduction steps

1. Navigate to any of web component charts mentioned above and try to access the inner contents using key board.
2.
3.
...

### Expected Behaviour

Web component chart functionality should behave as like UI5 chart components, there should be no limitations between these two components.

### UI5 Web Components for React Version

2.11.1

### UI5 Web Components Version

2.11.1

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @Muthu3982 

the linked PR will i.a. add arrow key navigation & accessibility attributes (if `chartConfig.accessibilityLayer` is active) to the `ScatterChart`. The `DonutChart` `accessibilityLayer` is implemented via this [PR](https://github.com/UI5/webcomponents-react/pull/8514).

Please note that the same points mentioned in [this comment](https://github.com/UI5/webcomponents-react/issues/7609#issuecomment-4336697385) apply here as well.
